Makefile

```diff
@@ -23,3 +23,9 @@ DIRS += modules
 modules_DEPEND_DIRS = src
 
 include $(TOP)/configure/RULES_TOP
+
+readthedocs: inc doc
+	$(MAKE) -C documentation/ doxygen T_A=any
+	$(PYTHON) -m sphinx documentation _readthedocs/html
+
+.PHONY: readthedocs
```

Comment:
The EPICS build rules get re-used by every downstream module, and we try to provide rules that can be used by them too. These targets really belong in the `configure/RULES_TOP` file to allow that.

Reply:
I'm not sure we should expose these rules, they are quite specific to the way epics-base is architectured. Maybe we can refactor then if and when people ask for it?

Reply:
I agree with @minijackson ; I think these rules would be of very limited, if any, use for downstream modules.

Reply:
Where `RULES_TOP` is concerned, downstream also includes the pvData, pvAccess and other external Base submodules; are there plans to eventually run Sphinx and RTD on them (which I think would have to happen in a separate stand-alone build, not when checked out as a submodule like this build is)?

I'm fine with the idea of refactoring the rules for future releases, but I disagree that the change would have limited use downstream. External support modules such as [Asyn](https://github.com/epics-modules/asyn/blob/master/docs/Makefile) and [AreaDetector](https://github.com/areaDetector/areaDetector/blob/master/docs/Makefile) are already using Sphinx, but have developed their own rules because they added them before Base did.

Reply:
> ... I'm fine with the idea of refactoring the rules for future releases ...

Would this include incompatible changes, and eventual removal?

I am also skeptical of the desirability of expanding the already expansive interface of our Makefiles.  At least not right away.  Also, I hesitate to enshrine `readthedocs` as a special name in a public interface.  There are alternatives, and RTD won't be around forever.

Reply:
I don't think this rule can be moved to `documentation/Makefile`, unfortunately. The RTD build needs the header files and Markdown files generated from `.pod` files.